react/src/components/Chat/EndpointLLMChatCard.tsx

```diff
@@ -93,37 +99,64 @@ const EndpointLLMChatCard: React.FC<EndpointLLMChatCardProps> = ({
 
   const submitId = useId();
 
+  const { agents } = useAgents();
+  const [agentId, setAgentId] = useState(defaultAgentId);
+  const agent = useMemo(() => {
+    return agents.find((a) => a.id === agentId);
+  }, [agents, agentId]);
+  const [experimentalChatAgent] = useBAISettingUserState(
+    'experimental_chat_agent',
+  );
+
   return (
     <LLMChatCard
       {...cardProps}
+      chatId={`${endpoint?.endpoint_id}_${agentId}`}
       baseURL={
         endpoint?.url
           ? new URL(basePath, endpoint?.url ?? undefined).toString()
           : undefined
       }
       models={models}
+      systemPrompt={agent?.config?.systemPrompt}
```

Comment:
`system_prompt` instead of `systemPrompt`? :)

Reply:
my bad, it also cause undefined value in systemPrompt. I will fix it